cmd/trade.go

```diff
@@ -331,14 +333,15 @@ func makeStrategy(
 	options inputs,
 	threadTracker *multithreading.ThreadTracker,
 	db *sql.DB,
+	tracker *metrics.MetricsTracker,
```

Comment:
can we call this variable `metricsTracker` -- I thought we had changed this yesterday.